src/test/java/uk/gov/hmcts/darts/cases/service/impl/CaseServiceImplTest.java

```diff
@@ -106,6 +107,8 @@ class CaseServiceImplTest {
 
     @Mock
     TranscriptionRepository transcriptionRepository;
+    @Mock
+    TranscriptionDocumentRepository transcriptionDocumentRepository;
```

Comment:
Should add some verifications on where this is used

Reply:
There aren't really any appropriate places to do this here, better coverage for this will be included in `HearingEntityToCaseHearingTest`